codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/localservice/nameresolver/SmithyNameResolver.java

```diff
@@ -30,6 +25,13 @@ public class SmithyNameResolver {
       "bool" //boolean shape
     )
   );
+  private static String awsServiceClientName = "Client";
+
+  public static Boolean isShapeFromAWSSDK(final Shape shape) {
+    // Is there a better way to do this?
+    // I did not figure out any other then hardcoding.
```

Comment:
Me neither.

Reply:
It's also likely that this has been lifted directly from the Python codegen :)